.github/ISSUE_TEMPLATE/New_feature_request.md

```diff
@@ -0,0 +1,21 @@
+---
+name: New Feature Request
+about: Request for engineering work on a new feature
+title: 'New Feature Request'
+labels: 
```

Comment:
We should add one of the existing labels that GitHub defines (e.g., `enhancement`)